api/scenario_api/include/scenario_api/scenario_api_core.h

```diff
@@ -283,25 +280,6 @@ class ScenarioAPI
   //****************************************************************************************************public API
 
 private:
-  ros::NodeHandle nh_;         //!< @brief ros node handle
-  ros::NodeHandle pnh_;        //!< @brief private ros node handle
-  ros::ServiceClient client_;  //!< @brief private ros service client
-  ros::Subscriber sub_state_;  //!< @brief topic subscriber for autoware state
-  ros::Subscriber sub_pcl_;    //!< @brief topic subscriber for pcl
-  ros::Subscriber sub_map_;    //!< @brief topic subscriber for map
-  ros::Subscriber
-    sub_route_;  //!< @brief topic subscriber for current route (for check of autoware ready)
-  ros::Subscriber sub_twist_;          //!< @brief topic subscriber for twist
-  ros::Subscriber sub_turn_signal_;    //!< @brief topic subscriber for turn signal(blinker)
-  ros::Timer timer_control_;           //!< @brief timer for getting self-position
-  ros::Publisher pub_start_point_;     //!< @brief topic pubscriber for start point
-  ros::Publisher pub_goal_point_;      //!< @brief topic pubscriber for goal point
```

Comment:
Where did all those go?

Reply:
they were declared but not defined. Looked like something was started but not finished. So I removed them